README.md

```diff
@@ -1,116 +1,101 @@
+## Development Requirements
 
-### Branches
+At the very least, you'll need to install a couple of tools to be able to build and run
+the site locally and to test your changes.
 
-The 'master' branch should be targeted for the majority of PRs.
+The following tool(s) must be installed on your system prior to developing the website:
 
-This repo has 'release branches' (i.e. `release-0.12`, `release-0.13`, ...).
-These are kept in-line with the release branches in the
-[`jetstack/cert-manager`](https://github.com/jetstack/cert-manager) repository.
+* cURL
+* NodeJS, version 16+
+* Golang, version 1.17+
 
-Once your pull request has been merged into master, it is typically
-'cherrypicked' into whatever release branches it is appropriate to make the
-change in.
+We also assume you've got GNU coreutils installed, which is usually the case by default on Linux
+and should be installable via Homebrew on macOS.
 
-If a change **only** affects a specific older version, pull requests can be
-opened directly against the documentation content in the `contents/docs`
-directory.
+## Website Development
 
-Changes to other areas of the codebase on release branches are not permitted.
+### Development Server
 
-The HTML/CSS/JavaScript used for the final website will always be deployed
-from the latest 'release branch' - this means that all versions of our
-documentation have a consistent style, and avoids us having to run the
-`hugo` command from multiple different release branches.
+Running a development server with hot-reload functionality is a one-liner:
 
-### Multi-versioning
-
-Because we publish documentation for multiple versions of cert-manager, we have
-an additional 'release' mode that can be used when building, developing or
-deploying the website.
-
-The majority of scripts mentioned below have corresponding `*-release` versions
-that can be used. These scripts will first gather all content from all the
-configured documentation versions and include them in the output.
-
-This is **not** enabled by default as it requires running `git clone` multiple
-times against multiple copies of the repository, which is a time consuming
-operation.
+```bash
+./scripts/server
+```
 
-### Requirements
+This script will run `npm install` and then start a development server.
 
-The majority of tools needed to build the website are automatically installed
-and versioned when you build the site by the tools in `scripts/`.
+If you've already run `npm install`, you can manually run `npm run dev` as another option.
 
-You also need to ensure you have the following tool(s) installed on your system
-as they are not managed by the `scripts/` directory:
+In any case, a server will spin up at `http://localhost:3000`.
 
-* npm/nodejs
-* Golang 1.12+
+Initial builds of a page on the development server can be quite slow - a few seconds - but
+after the initial build changes should be picked up quickly and the development server
```

Comment:
after the initial build, changes should be picked up quickly and the development server

Reply:
Yeah would maybe be better with the comma but probs not worth changing; thanks for the suggestion though!